### DISCUSSION TITLE
DOSXYZnrc [plot profile]

### DISCUSSION THREAD

**Discussion by Mona99-del** [nrc-cnrc/EGSnrc#1169] [General]
![image](https://github.com/user-attachments/assets/f61c00c2-8a20-4235-874d-340cb5757080)
When trying to plot profile after completing the simulation, this error appears as shown in the picture. Do you know how I can solve it? This information file demo.egslst. Can you help me?
NRCC/UW EGSnrc user-code DOSXYZnrc
 ON win3264                                                16:09:59 Aug 18 2024
 *******************************************************************************
 **                                                                           **
 **                                  DOSXYZnrc                                **
 **                              Z pronounced zed                             **
 **                                                                           **
 **      Code developed at the National Research Council of Canada and        **
 **           University of Wisconsin as part of the OMEGA project            **
 **                                                                           **
 **                                                                           **
 **                                                                           **
 *******************************************************************************

     The following parameters may be adjusted in dosxyz_user_macros.mortran
 $MXMED:    Max number of media: 20
 $MXSTACK:  Max stack size:    900000
 $IMAX,etc: Max dose scoring regions in x,y,z directions:  128  128  128
 $MAXDOSE:  Max dose scoring regions consistent with above:2097153
 $DOSEZERO(=1) 1=> all doses with uncert > 50% are zeroed in .3ddose file

 The following parameters may be adjusted in srcxyz.macros
 $INVDIM:   number of elements in inverse CPD for input energy spectra = 1000
 $NENSRC:   number of bins in input energy spectrum =  200

 ===============================================================================
 -------------------------------------------------------------------------------

 Title:  6Mv 10cm*10cm                                                                   
 -------------------------------------------------------------------------------
 ===============================================================================

 Number of media (min = 1, max =  20, 0 => CT data):                2
 Medium  1:                     H2O700ICRU              
 Medium  2:                     AIR700ICRU              

 ECUTIN,PCUTIN,(ESTEPE,SMAX--DUMMY INPUTS): 
              0.000     0.000     0.000     0.000     0.000

 # regions in x (max= 128),y (max= 128),z (max= 128) directions
 (if<0,implies # groups of reg), IPHANT (1 to output a .egsphant
 file for dosxyz_show, 0[default] to not output this file)
     :      -3    -3    -2     1

 Input boundaries in the x-direction
 -----------------------------------
 Initial boundary:       -15.000
 Width in this group, number of regions in group:         1.000    5
 Width in this group, number of regions in group:         0.200  100
 Width in this group, number of regions in group:         1.000    5
 Boundaries
     -15.000     -14.000     -13.000     -12.000     -11.000     -10.000
      -9.800      -9.600      -9.400      -9.200      -9.000      -8.800
      -8.600      -8.400      -8.200      -8.000      -7.800      -7.600
      -7.400      -7.200      -7.000      -6.800      -6.600      -6.400
      -6.200      -6.000      -5.800      -5.600      -5.400      -5.200
      -5.000      -4.800      -4.600      -4.400      -4.200      -4.000
      -3.800      -3.600      -3.400      -3.200      -3.000      -2.800
      -2.600      -2.400      -2.200      -2.000      -1.800      -1.600
      -1.400      -1.200      -1.000      -0.800      -0.600      -0.400
      -0.200      -0.000       0.200       0.400       0.600       0.800
       1.000       1.200       1.400       1.600       1.800       2.000
       2.200       2.400       2.600       2.800       3.000       3.200
       3.400       3.600       3.800       4.000       4.200       4.400
       4.600       4.800       5.000       5.200       5.400       5.600
       5.800       6.000       6.200       6.400       6.600       6.800
       7.000       7.200       7.400       7.600       7.800       8.000
       8.200       8.400       8.600       8.800       9.000       9.200
       9.400       9.600       9.800      10.000      11.000      12.000
      13.000      14.000      15.000

 Input boundaries in the y-direction
 -----------------------------------
 Initial boundary:       -15.000
 Width in this group, number of regions in group:         1.000    5
 Width in this group, number of regions in group:         0.200  100
 Width in this group, number of regions in group:         1.000    5
 Boundaries
     -15.000     -14.000     -13.000     -12.000     -11.000     -10.000
      -9.800      -9.600      -9.400      -9.200      -9.000      -8.800
      -8.600      -8.400      -8.200      -8.000      -7.800      -7.600
      -7.400      -7.200      -7.000      -6.800      -6.600      -6.400
      -6.200      -6.000      -5.800      -5.600      -5.400      -5.200
      -5.000      -4.800      -4.600      -4.400      -4.200      -4.000
      -3.800      -3.600      -3.400      -3.200      -3.000      -2.800
      -2.600      -2.400      -2.200      -2.000      -1.800      -1.600
      -1.400      -1.200      -1.000      -0.800      -0.600      -0.400
      -0.200      -0.000       0.200       0.400       0.600       0.800
       1.000       1.200       1.400       1.600       1.800       2.000
       2.200       2.400       2.600       2.800       3.000       3.200
       3.400       3.600       3.800       4.000       4.200       4.400
       4.600       4.800       5.000       5.200       5.400       5.600
       5.800       6.000       6.200       6.400       6.600       6.800
       7.000       7.200       7.400       7.600       7.800       8.000
       8.200       8.400       8.600       8.800       9.000       9.200
       9.400       9.600       9.800      10.000      11.000      12.000
      13.000      14.000      15.000

 Input boundaries in the z-direction
 -----------------------------------
 Initial boundary:         0.000
 Width in this group, number of regions in group:         0.100  100
 Width in this group, number of regions in group:         1.000   10
 Boundaries
       0.000       0.100       0.200       0.300       0.400       0.500
       0.600       0.700       0.800       0.900       1.000       1.100
       1.200       1.300       1.400       1.500       1.600       1.700
       1.800       1.900       2.000       2.100       2.200       2.300
       2.400       2.500       2.600       2.700       2.800       2.900
       3.000       3.100       3.200       3.300       3.400       3.500
       3.600       3.700       3.800       3.900       4.000       4.100
       4.200       4.300       4.400       4.500       4.600       4.700
       4.800       4.900       5.000       5.100       5.200       5.300
       5.400       5.500       5.600       5.700       5.800       5.900
       6.000       6.100       6.200       6.300       6.400       6.500
       6.600       6.700       6.800       6.900       7.000       7.100
       7.200       7.300       7.400       7.500       7.600       7.700
       7.800       7.900       8.000       8.100       8.200       8.300
       8.400       8.500       8.600       8.700       8.800       8.900
       9.000       9.100       9.200       9.300       9.400       9.500
       9.600       9.700       9.800       9.900      10.000      11.000
      12.000      13.000      14.000      15.000      16.000      17.000
      18.000      19.000      20.000

 Total # regions including exterior =   1331001

 Input groups of regions for which density and medium are not defaults
 Lower,upper i, j, k,  MEDIUM, DENSITY (  1 110)(  1 110)(  1 110)   1     0.000
 Lower,upper i, j, k,  MEDIUM, DENSITY ( 46  65)( 51  60)( 11  20)   2     0.000
 Lower,upper i, j, k,  MEDIUM, DENSITY
    Found blank line => end of this input

 Input groups of regions for which ECUT and PCUT are not defaults
 NB This option is disabled, just input 8 zeros.
 Dummy values of lower,upper i, j, k,  ECUT, PCUT
    Found blank line => end of this input

 Enter 8 numbers on one line
   3 pairs defininglower,upper x,y,z indicies of dose regions
                 for which results are to be output
   IZSCAN:       non-zero for z-scan/page
   MAX20:        if any one = 1, output summary of max 20 doses.
   end signaled by first pair both zero
   forno dose printed, MAX20 is still read from first line

    Found blank line => end of this input
 No doses will be output to egslst file

 The material in the region outside the phantom is:AIR700ICRU              
 The thickness of this region (in x, y & z direction) is:  50.000 cm

 NCASE,IWATCH,TIMMAX,INSEED1,INSEED2,BEAM_SIZE,ISMOOTH,IRESTART,IDAT,
 IREJECT,ESAVE_GLOBAL,NRCYCL,IPARALLEL,PARNUM,n_split,ihowfarless,i_phsp_out,
 i_bindos 
 : 
     6000000   0 500.00        33        97 100.00      0    0   0   0   0.00   0   0   0   1   0   0

================================================================================

                   Electron/Photon transport parameter

================================================================================

 Photon cross sections                                      xcom
 Compton cross sections                                     default
 Photon transport cutoff(MeV)                                    0.1000E-01
 Pair angular sampling                                       SIM
 Pair cross sections                                         BH 
 Triplet production                                          Off
 Bound Compton scattering                                    OFF           
 Radiative Compton corrections                               Off           
 Rayleigh scattering                                         OFF           
 Atomic relaxations                                          OFF           
 Photoelectron angular sampling                              OFF           

 Electron transport cutoff(MeV)                               0.7000
 Bremsstrahlung cross sections                              BH  
 Bremsstrahlung angular sampling                             SIM
 Spin effects                                                On
 Electron Impact Ionization                                  Off
 Maxium electron step in cm (SMAX)                               0.1000E+11
 Maximum fractional energy loss/step (ESTEPE)                0.2500
 Maximum 1st elastic moment/step (XIMAX)                     0.5000
 Boundary crossing algorithm                                 EXACT     
 Skin-depth for boundary crossing (MFP)                      3.000    
 Electron-step algorithm                                     PRESTA-II 

================================================================================

 

 *******************************************************************************

         Summary of source parameters (srcxyznrc)
 *******************************************************************************

               BEAM treatment head simulation used as source:

          x-coordinate of the isocenter,                         0.0000 cm
          y-coordinate of the isocenter,                         0.0000 cm
          z-coordinate of the isocenter,                         0.0000 cm
          Polar angle of beam axis:                            180.0000 degrees
          Azimuthal angle of beam axis:                          0.0000 degrees
          Distance from isocenter to centre of scoring plane:   -0.0000 cm
          Rotation angle about beam axis,                        0.0000 degrees
          BEAM simulation: BEAM_EX16MVp
               input file: demo
           pegs data file: 700icru

          Fat photons from DBS will not be rejected.

 Particles to be simulated: all

 Medium                AE        AP
 H2O700ICRU          0.700     0.010
 AIR700ICRU          0.700     0.010

 No range rejection.

 ***************************************************************

  Histories to be simulated for this run      6000000

  Histories to be analyzed after this run     6000000

 ***************************************************************
   Elapsed wall clock time to this point=       1.985 s

   CPU time so far for this run =       1.062 s

 BATCH #  TIME-ELAPSED  TOTAL CPUTIME  RATIO  TIME OF DAY  RNG pointers

     1          0.0            0.0      0.00    16:10:01   ixx jxx =   97  33 
     2       6793.4         6732.4      1.01    18:03:14   ixx jxx =   33  66 
     3      13737.5        13570.5      1.01    19:58:58   ixx jxx =   60  93 
     4      20599.4        20337.0      1.01    21:53:20   ixx jxx =   65   1 
     5      27814.4        27279.4      1.02    23:53:35   ixx jxx =    7  40 
     6      34748.5        34085.2      1.02    01:49:09   ixx jxx =    8  41 
     7      42278.5        41308.9      1.02    03:54:39   ixx jxx =   41  74 
     8      49056.1        48080.7      1.02    05:47:37   ixx jxx =   43  76 
     9      55776.1        54799.9      1.02    07:39:37   ixx jxx =   62  95 
    10      62497.3        61519.8      1.02    09:31:38   ixx jxx =   21  54 

 Total CPU time for run = 68239.0 s =  18.955 hr =>     316535. hist/hr
   On  win3264 

  ********************************************************************

 # of particles read from BEAM simulation (N_read)              =     6000000
 # of primary particles incident in BEAM simulation             =   345512880
 # of particles discarded dueto charge/LATCH/W/multiple passer =           0
 # of particles discarded because beyond BEAM_SIZE              =           0
 # of photons rejected because beyond DBS splitting radius      =           0
 # of particles that missed geometry                            =           0
 # of particles simulated in DOSXYZnrc (N_used)                 =     6000000
                        N_used/N_read       =       1.000

  ********************************************************************

 Fraction of incident energy deposited in the phantom =      0.3919

 Fraction of incident energy deposited in the region surrounding
 the phantom when incident particles go through it   =      0.0010

 Number of charged particle steps simulated,   N_step   =      543280982
 Number of charged particle steps/incident fluence      =    1.57239E+00
 No. of PRESTA-II steps/total no. of charged particle steps =        0.56934

1

 The dose values in     798005 voxels had error > 50% and have been
 zeroed in the .3ddose file.

 Total CPU time for this run = 68253.8 s =  18.959 hr

 END OF RUN          Aug 19 2024 11:23:55

**Comment by rtownson** [comment]
The error you're seeing from xmgrace is just to do with the plotting, nothing concerning. It's just adjusting your y-axis since all the numbers are squished together.

**Comment by Mona99-del** [comment]
Thanks for your reply how can i adjust y-axis؟

**Reply by rtownson** [reply]
Probably under Plot->Axis properties. Or just copy/paste the data into a plotting/spreadsheet tool that you're familiar with.

**Reply by Mona99-del** [reply]
ok, thank you so much

**Comment by Mona99-del** [comment]
![image](https://github.com/user-attachments/assets/6ec5c12b-d7ce-4d20-b737-7feb62373754)
![image](https://github.com/user-attachments/assets/1372070d-8f80-4501-b958-22826e7bbacb)
![image](https://github.com/user-attachments/assets/e7165e95-84bd-49c6-9f8a-6abe87d85871)

I tried using VICTORIA and this is the result is the simulation correct?

**Reply by rtownson** [reply]
It looks reasonable to me, if you wanted an air pocket in a water box. The uncertainties look pretty large.

**Comment by Mona99-del** [comment]
ok, thank you for the replay. if I put another material instead of air,  such as silicone, will reduce the uncertainty?

**Reply by rtownson** [reply]
Denser materials will have more interactions, so yes there would be a lower uncertainty on the dose deposited. But the best way to reduce the uncertainty is just to run longer simulations of more particles.

**Comment by Mona99-del** [comment]
Ok, thanks for the clarification.
